src/contracts/pods/EigenPod.sol

```diff
@@ -102,7 +100,7 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
     event ValidatorBalanceUpdated(uint40 validatorIndex, uint64 newValidatorBalanceGwei);
     
     /// @notice Emitted when an ETH validator is prove to have withdrawn from the beacon chain
-    event FullWithdrawalRedeemed(uint40 validatorIndex, address indexed recipient, uint64 withdrawalAmountGwei);
+    event FullWithdrawalRedeemed(uint40 validatorIndex, address indexed recipient, uint256 withdrawalAmountWei);
```

Comment:
probably use `pubkeyHash` instead of `validatorIndex` here in order to adapt to indices being reused in the future

Reply:
@Sidu28 this is still an open item. 
sounds like there's an existing EIP that would allow reuse of validator indices
seems best to change what we're emitting in events so we would be compatible with this